packages/model-viewer/src/test/features/scene-graph/mesh-primitive-spec.ts

```diff
@@ -0,0 +1,134 @@
+/* @license
+ * Copyright 2020 Google LLC. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {MeshStandardMaterial} from 'three/src/materials/MeshStandardMaterial';
+
+import {$primitives, Model} from '../../../features/scene-graph/model.js';
+import {CorrelatedSceneGraph} from '../../../three-components/gltf-instance/correlated-scene-graph.js';
+import {assetPath, loadThreeGLTF} from '../../helpers.js';
+
+
+
+const expect = chai.expect;
+
+const BRAIN_STEM_GLB_PATH = assetPath(
+    'models/glTF-Sample-Models/2.0/BrainStem/glTF-Binary/BrainStem.glb');
+const SHEEN_CHAIR_GLB_PATH = assetPath(
+    'models/glTF-Sample-Models/2.0/SheenChair/glTF-Binary/SheenChair.glb');
```

Comment:
We might want to pick smaller test models; If they take a long time to load it'll slow down  the whole test pass. Not a huge deal, just keep an eye on how long the CI takes to pass.